main.go

```diff
@@ -93,13 +93,14 @@ func getConn(flags *flagStorage) (c *gcsx.Connection, err error) {
 
 	// Create the connection.
 	cfg := &gcs.ConnConfig{
-		Url:                 flags.Endpoint,
-		TokenSource:         tokenSrc,
-		UserAgent:           fmt.Sprintf("gcsfuse/%s %s", getVersion(), flags.AppName),
-		MaxBackoffSleep:     flags.MaxRetrySleep,
-		DisableHTTP2:        flags.DisableHTTP2,
-		MaxConnsPerHost:     flags.MaxConnsPerHost,
-		MaxIdleConnsPerHost: flags.MaxIdleConnsPerHost,
+		Url:                        flags.Endpoint,
+		TokenSource:                tokenSrc,
+		UserAgent:                  fmt.Sprintf("gcsfuse/%s %s", getVersion(), flags.AppName),
+		MaxBackoffSleep:            flags.MaxRetrySleep,
+		DisableHTTP2:               flags.DisableHTTP2,
+		MaxConnsPerHost:            flags.MaxConnsPerHost,
+		MaxIdleConnsPerHost:        flags.MaxIdleConnsPerHost,
+		EnableStorageClientLibrary: flags.EnableStorageClientLibrary,
```

Comment:
Is it possible for you to forwat the code using golangci-lint  ?

Reply:
Hey I used golangci-lint but it's not showing any suggestion to format this code piece.